language-design.md

```diff
@@ -102,6 +201,18 @@ you do so? If not, why not?
 
 **Response**
 
+We think that this shows that there is a balance to be struck between natural
+language design and regular scripting language syntax.  It is also important
+to know the audience of the language.  If they are experienced programmers it
+is probably easier for them if your programming language looks more like what
+they are used to and has the less natural syntax of Java or C, but if they are
+less experienced it would feel more natural to the user to use a more natural
+language type of syntax and grammer.
```

Comment:
I'm not convinced that when you run into a bug you'd use your natural language instincts to solve it (unless you are super, super, super new). That being said, for algorithmic issues I think it's easier to reason about it with a natural-ish syntax.